File: testdata/repos/commons/templates/create_formula/languages/php/helloWorld/src/formula/formula.php
<?php
use Codedungeon\PHPCliColors\Color;

function Run($input1, $input2, $input3, $input4) {
    echo "Hello World!\n";
    echo Color::GREEN, "My name is $input1.\n";

    if ($input2 === "true") {
        echo Color::BLUE, "I've already created formulas using Ritchie.\n";
    } else {
        echo Color::RED, "I'm excited in creating new formulas using Ritchie.\n";
    }
    echo Color::YELLOW, "Today, I want to automate $input3.\n";
    echo Color::CYAN, "My secret is $input4.\n";
}
?>

File: testdata/repos/commons/templates/create_formula/languages/csharp/helloWorld/src/formula/formula.cs
using System;

namespace formula
{
    public class Hello
    {
        public string input1;
        public string input2;
        public string input3;
        public string input4;
        public Hello(string intext, string inlist, string inbool, string insecret)
        {
            this.input1 = intext;
            this.input2 = inlist;
            this.input3 = inbool;
            this.input4 = insecret;

            Console.WriteLine("Hello World!");
            Console.ForegroundColor = ConsoleColor.Green;
            Console.WriteLine($"My name is {input1}.");
            Console.ForegroundColor = ConsoleColor.Blue;
            if (input3 == "true")   {
                Console.WriteLine("I've already created formulas using Ritchie.");
            } else  {
                Console.WriteLine("I'm excited in creating new formulas using Ritchie.");
            }
            Console.ForegroundColor = ConsoleColor.Yellow;
            Console.WriteLine($"Today, I want to automate {input2}.");
            Console.ForegroundColor = ConsoleColor.Cyan;
            Console.WriteLine($"My secret is {input4}.");
            Console.ResetColor();
        }

    }
}
